tests/raftstore_cases/test_conf_change.rs

```diff
@@ -638,6 +638,12 @@ fn test_learner_conf_change<T: Simulator>(cluster: &mut Cluster<T>) {
     pd_client.transfer_leader(r1, new_peer(4, 12));
     pd_client.region_leader_must_be(r1, new_peer(4, 12));
 
+    // Transfer leader to (1, 1) to avoid "region not found".
+    pd_client.transfer_leader(r1, new_peer(1, 1));
```

Comment:
Why move it here? What's the point of L638 then?

Reply:
L638 is for ensure after promoted, the field of `Peer::peer` must be with `is_learner` is false.

Reply:
Comment it then.